spec/factories/place_geometries.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :place_geometry do
+    place
+    source
```

Comment:
I'm surprised this works without a `geom` value, but I guess we can deal with that when it comes up.

Reply:
Yeah, I see that validation now. I think it's only used right now in the context of creating a place (`after(:build) { |place| build :place_geometry, place: place }` ), but only built, not persisted. I'll see what we can put in place for full creation.